.github/workflows/ros-workflow.yml

```diff
@@ -99,7 +99,7 @@ jobs:
 
     - name: Mobtest validations
       run: |
-        mobtest $(pwd)
+        mobtest --sonar_issues_path ./mobtest-issues.json .
```

Comment:
if that json file does not exist, does mobtest blow up?

Reply:
That is the file to which mobtest writes the results, it doesn't matter if exists or not :)